repo/packages/M/minio/0/marathon.json.mustache

```diff
@@ -0,0 +1,62 @@
+{
+    "id": "{{service.name}}",
+    "cpus": {{service.cpus}},
+    "mem": {{service.mem}},
+    "instances": 1,
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.minio-docker-RELEASE.2016-09-11T17-42-18Z}}",
```

Comment:
This name shouldn't have a `.` in it. The mustache rendering code thinks that is a sub object. So when it renders the template it will traverse the json document as if it is:
```json
{
  "resource": {
    "asserts": {
      "container": {
        "docker": {
          "minio-docker-RELEASE": {
            "2016.09-11T17-42-18Z": "minio/minio:RELEASE.2016-09-11T17-42-18Z"
          }
        }
      }
    }
  }
}
```

Instead of what you have listed here. 

You can pick any name for the image so it instead could be `resource.assets.container.docker.minio`